src/react-components/room/ContentMenu.js

```diff
@@ -7,9 +7,9 @@ import { ReactComponent as ObjectsIcon } from "../icons/Objects.svg";
 import { ReactComponent as PeopleIcon } from "../icons/People.svg";
 import { FormattedMessage } from "react-intl";
 
-export function ContentMenuButton({ active, children, ...props }) {
+export function ContentMenuButton({ active, disabled, children, ...props }) {
```

Comment:
You'll wanted to add disabled to the proptypes on line 18. Ideally we want to move to Typescript to solve this, but this is a pretty big file so not sure its worth it for this project. PropType should suffice for now.
eg.
`disabled: PropTypes.bool,`

Reply:
Amazing! Thanks for catching this as well!